packages/stories-block/src/block/components/storyPicker/storyPicker.js

```diff
@@ -119,7 +119,12 @@ function StoryPicker({
 
   return (
     <Modal
-      title={__('Selected Stories', 'web-stories')}
+      title={_n(
+        'Selected Story',
+        'Selected Stories',
+        maxNumOfStories,
+        'web-stories'
+      )}
```

Comment:
`_n()` should only be used if the string contains a number/placeholder.

If not, need to do `1 === maxNumOfStories ? _n('Selected Story', 'web-stories') : _n('Selected Stories', 'web-stories')`